### ISSUE TITLE
[feature request] abspath / normpath for PATH

### ISSUE THREAD

**Issue by vadimkantorov** [emscripten-core/emscripten#13145]
These are very useful

**Comment by sbc100** [comment]
I think you should be able to use the standard C++ std::filesystem module for this.   See https://en.cppreference.com/w/cpp/filesystem/absolute

**Comment by vadimkantorov** [comment]
I'm coding in browser JavaScript, and use PATH as my file path API. It already has basename/dirname/join2, so abspath/normpath may make sense

**Comment by sbc100** [comment]
Oh, you mean that PATH object defined in src/library_path.js?  

It looks like it already has `PATH.normalize` which is probably what you want for `normpath`?  

I'm not sure about `abspath`..  if you (or someone) would like to add it that sounds reasonable.

**Comment by vadimkantorov** [comment]
> Oh, you mean that PATH object defined in src/library_path.js?

Yeah, that's what I meant :)

**Comment by vadimkantorov** [comment]
Yep, normalize does the trick. I guess having a abspath/normpath alias is what's left of this request
